### PR TITLE
Documentation for Sony Bravia TV with Pre-Shared Key

### PR DESCRIPTION
**Pull request by gerard33**
**Description:**
This is the documentation for the new component for Sony Bravia TV with Pre-Shared Key.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#12464

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/media_player.braviatv_psk.markdown [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    psk: sony
+    mac: AA:BB:CC:DD:EE:FF
+    amp: True
+    android: False
+    sourcefilter:
+      - ' HD'
+      - HDMI
+```
+
+**Configuration variables**
```
Comment (c727) [inline source/_components/media_player.braviatv_psk.markdown]: check {% configuration %} from here: https://raw.githubusercontent.com/home-assistant/home-assistant.github.io/current/source/_components/input_boolean.markdown

File: source/_components/media_player.braviatv_psk.markdown [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+- **name** (*Optional*): The name used on the frontend, defaults to Sony Bravia TV
+- **host** (*Required*): The IP of the Sony Bravia TV, eg. 192.168.1.191
+- **psk** (*Required*): The Pre-Shared Key of the Sony Bravia TV, eg sony. Make sure that you place the psk between quotes when it has a leading zero (eg 0000 should be in the config file as '0000')
+- **mac** (*Optional*): The MAC address of the Sony Bravia TV, eg. AA:BB:CC:DD:EE:FF. This is only required for TVs without Android software and is needed to remotely start the TV
+- **amp** (*Optional*): Boolean, defaults to False. Set this to True if you have an amplifier attached to the TV and not use the internal TV speakers. Then the volume slider will not be shown as this doesn’t work for the amplifier. Mute and volume up and down buttons are there and working with an amplifier.
+- **android** (*Optional*): Boolean, defaults to True as all new TV models have Android software. Set this to False when you don't have an Android TV as these TV’s only respond to WakeOn LAN commands via the MAC address.
```
Comment (c727) [inline source/_components/media_player.braviatv_psk.markdown]: TVs

File: source/_components/media_player.braviatv_psk.markdown [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+- **sourcefilter** (*Optional*): List of text that is used to filter the source list, eg. ' HD' (with quotes) will only show TV channels in the source list which contain ' HD', eg. 'NPO 3 HD'.
+
+**Installation instructions TV**
+You will need to configure your TV to allow Home Assistant for remote usage by performing the following steps.
+
+- Enable remote start on your TV: [Settings] -> [Network] -> [Home Network Setup] -> [Remote Start] -> [On]
+- Enable pre-shared key on your TV: [Settings] -> [Network] -> [Home Network Setup] -> [IP Control] -> [Authentication] -> [Normal and Pre-Shared Key]
+- Set pre-shared key on your TV: [Settings] -> [Network] -> [Home Network Setup] -> [IP Control] -> [Pre-Shared Key] -> sony
+- Give your TV a static IP address, or make a DHCP reservation for a specific IP address in your router
+- For non-Android TV's: determine the MAC address of your TV: [Settings] -> [Network] -> [Network Setup] -> [View Network Status]
```
Comment (c727) [inline source/_components/media_player.braviatv_psk.markdown]: TVs

File: source/_components/media_player.braviatv_psk.markdown [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+comments: false
+sharing: true
+footer: true
+logo: bravia.png
+ha_category: Media Player
+ha_release: 0.64
+ha_iot_class: "Local Polling"
+---
+
+The `braviatv_psk` platform allows you to control a [Sony Bravia TV](http://www.sony.com). This version used a Pre-Shared Key (PSK) for the connection between HA and your TV. The PSK can be configured in the TV settings.
```
[the remark this answers is not in the document]
Comment (gerard33) [reply source/_components/media_player.braviatv_psk.markdown]: Done

File: source/_components/media_player.braviatv_psk.markdown [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+Almost all [Sony Bravia TV from 2014 and newer](http://info.tvsideview.sony.net/en_ww/home_device.html#bravia) are supported.
```
[the remark this answers is not in the document]
Comment (gerard33) [reply source/_components/media_player.braviatv_psk.markdown]: Done

File: source/_components/media_player.braviatv_psk.markdown [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+    sourcefilter:
+      - ' HD'
+      - HDMI
+```
+
+**Configuration variables**
+
+- **name** (*Optional*): The name used on the frontend, defaults to Sony Bravia TV
+- **host** (*Required*): The IP of the Sony Bravia TV, eg. 192.168.1.191
+- **psk** (*Required*): The Pre-Shared Key of the Sony Bravia TV, eg sony. Make sure that you place the psk between quotes when it has a leading zero (eg 0000 should be in the config file as '0000')
```
[the remark this answers is not in the document]
Comment (gerard33) [reply source/_components/media_player.braviatv_psk.markdown]: Done

File: source/_components/media_player.braviatv_psk.markdown (hunk 1 above)
Comment (gerard33) [reply source/_components/media_player.braviatv_psk.markdown]: Done

File: source/_components/media_player.braviatv_psk.markdown (hunk 2 above)
Comment (gerard33) [reply source/_components/media_player.braviatv_psk.markdown]: Done

File: source/_components/media_player.braviatv_psk.markdown (hunk 3 above)
Comment (gerard33) [reply source/_components/media_player.braviatv_psk.markdown]: Done.
Thanks for the review, I have updated all your requested changes.

**Comment by gerard33** [comment]
Closing this PR as the goal is to merge this new component with the existing braviatv component.

**Pull request closed without merging**